doc/TRAINING.rst

```diff
@@ -224,6 +229,26 @@ Note: the released models were trained with ``--n_hidden 2048``\ , so you need t
 
    Key cudnn_lstm/rnn/multi_rnn_cell/cell_0/cudnn_compatible_lstm_cell/bias/Adam not found in checkpoint
 
+
+Transfer-Learning (new alphabet)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you want to train a DeepSpeech model for a new language, or if you just want to add new characters to your custom English model, you will have to use transfer-learning. Transfer-learning for DeepSpeech works by removing the output layer (any optionally more layers) of a pre-trained model and reinitializing parameters to fit the target alphabet. The simplest case of transfer-learning is when you remove just the output layer.
+
+To use transfer-learning, you need to specify the location of the pre-trained model with ``--load_checkpoint_dir`` and define where your new model checkpoints will be saved with ``--save_checkpoint_dir``. Now, you need to specify how many layers to remove (aka "drop") from the pre-trained model: ``--drop_source_layers``. You also need to supply a new alphabet file using the standard ``--alphabet_config_path``.  in ``DeepSpeech.py``. Specify the path where you downloaded the checkpoint from the release, and training will resume from the pre-trained model.
```

Comment:
Can you explain a bit more about the behavior of `--drop_source_layers` ?

Reply:
done